gotpl/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang
```

Comment:
We should tag this to a specific tag of golang.

Reply:
Fair enough